source/main.py

```diff
@@ -108,6 +108,10 @@ def main():
     if options["subcommand"] in ("run", "verify"):
         workdir = misc.create_workdir(start_ts, conf, logger.logger)
         Loggers.set_logging_to_file(logger, workdir, start_ts, conf)
+        logger.logger.handlers[0].setTarget(logger.logger.handlers[2])
```

Comment:
what's handlers[2]? Need at least comment on that to avoid issues from other changes - is that vulnerable to loggers definition order?